mod yajl: compile issues on FreeBSD

On FreeBSD 10.3 module yajl does not compile.
I think `testmod_yajl` is missing the `${YAJL_INCLUDE_DIR}` while the plugin itself is compiled with the proper `INCLUDE_DIRECTORIES`.

```
--- src/plugins/yajl/CMakeFiles/testmod_yajl.dir/all ---
In file included from /home/mpranj/workspace/libelektra/src/plugins/yajl/testmod_yajl.c:11:
/home/mpranj/workspace/libelektra/build/src/plugins/yajl/yajl.h:49:11: fatal error: 'yajl/yajl_version.h' file not found
```

--- a/src/plugins/yajl/CMakeLists.txt
+++ b/src/plugins/yajl/CMakeLists.txt
@@ -10,7 +10,10 @@ if (DEPENDENCY_PHASE)
 		install(DIRECTORY yajl DESTINATION ${TARGET_TEST_DATA_FOLDER})
 
 		include_directories(${CMAKE_CURRENT_BINARY_DIR})
-		add_plugintest (yajl)
+		add_plugintest (yajl
+			INCLUDE_DIRECTORIES
+				${YAJL_INCLUDE_DIR}
+			)
 	else (YAJL_FOUND)
 		remove_plugin (yajl "yajl not found")
 	endif (YAJL_FOUND)

